src/App.tsx

```diff
@@ -1,14 +1,73 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+  useCallback,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+const getAll = async (): Promise<Todo[]> => {
+  return getTodos();
+};
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoad, setIsLoad] = useState(true);
+  const [isModalOpen, setModalOpen] = useState(false);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [selectValue, setSelectValue] = useState<string>('all');
+  const [inputValue, setInputValue] = useState<string>('');
```

Comment:
it's unnecessary types, TS already knows about that, bc you put an initial value
```suggestion
  const [selectValue, setSelectValue] = useState('all');
  const [inputValue, setInputValue] = useState('');
```